deployments/helm/nvidia-dra-driver-gpu/templates/_helpers.tpl

```diff
@@ -66,14 +66,19 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
 {{/*
-Selector labels
+Selector labels: meant to select just the pods of the corresponding
+deployment/daemonset/... That is, this is different per-component.
 */}}
 {{- define "nvidia-dra-driver-gpu.selectorLabels" -}}
-{{- if .Values.selectorLabelsOverride -}}
-{{ toYaml .Values.selectorLabelsOverride }}
-{{- else -}}
-{{ include "nvidia-dra-driver-gpu.templateLabels" . }}
-{{- end }}
+  {{- if and (hasKey . "componentName") (hasKey . "context") -}}
+    {{- if .context.Values.selectorLabelsOverride -}}
+    {{ toYaml .context.Values.selectorLabelsOverride }}
```

Comment:
Question: Since this is indented, is that going to mess with indentation in the rendered template?

Reply:
By the way: Not sure whether the following is useful: https://github.com/NVIDIA/k8s-device-plugin/blob/main/tests/helm/helm_template_test.go